subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -703,6 +703,60 @@ The TestNG framework has a quite similar concept. In TestNG you can specify diff
             </sample>
 ++++
 
+[[migrate_to_junit5]]
+==== Migrate to JUnit 5
+
+http://junit.org/junit5/docs/current/user-guide[JUnit 5] is the latest version of the well-known `JUnit` test framework.
```

Comment:
I don't think we need to make the word "JUnit" code-formatted.

Reply:
All Done.